docs/Formatting.md

```diff
@@ -6,29 +6,15 @@
 
 ***
 
-### xml.format.enforceQuoteStyle
-
-Enforce `preferred` quote style (set by `xml.preferences.quoteStyle`) or `ignore` quote style when formatting.
-
-For instance, when set to `preferred` with `xml.preferences.quoteStyle` set to `single`, the following document:
-
-  ```xml
-  <?xml version="1.0" encoding="UTF-8"?>
-  <root>
-    <child attribute='value' otherAttribute="otherValue"></child>
-  </root>
-  ```
-
-will be formatted to:
-
-  ```xml
-  <?xml version='1.0' encoding='UTF-8'?>
-  <root>
-    <child attribute='value' otherAttribute='otherValue'></child>
-  </root>
-  ```
+### xml.format.experimental
 
-No changes to quotes will occur during formatting if `xml.format.enforceQuoteStyle` is set to `ignore`.
+  As we have more and more issues with our current XML formatter, we decide to redo our formatter to try to fix them. This new formatter is in experimental state because it doesn't support all current formatter settings and it could have some bugs. To enable this experimental formatter you can set to true the settings xml.format.experimental. Once we will have enough good feedback,  and we will support all current formatting settings, we will replace the current formatter with the experimental formatter. Don't hesitate to create issues to give us feedback with this experimental formatter.
```

Comment:
A few grammar things:

... we have decided to redo...

... you can set the setting `xml.format.experimental` to true...

Once we have enough good feedback and we support all current formatting settings...

Reply:
fixed